alacritty/src/window.rs

```diff
@@ -151,6 +153,9 @@ impl Window {
             create_gl_window(window_builder.clone(), &event_loop, false, logical)
                 .or_else(|_| create_gl_window(window_builder, &event_loop, true, logical))?;
 
+        // Set Wayland client-side decorations theme
+        Window::set_wayland_theme(event_loop, windowed_context.window(), config);
```

Comment:
I think we should follow our X11 approach here and move call to `#[cfg(not(any(..))` and do event_loop.is_wayland() then call `set_wayland_theme`.

Reply:
We do.